Products/CMFPlone/profiles/default/controlpanel.xml

```diff
@@ -352,4 +352,17 @@
   >
     <permission>Inspect Relations</permission>
   </configlet>
+
+  <!-- profiles/default/controlpanel.xml -->
+  <configlet action_id="plone-recyclebin"
+             appId="plone-recyclebin"
+             category="plone-general"
+             condition_expr=""
+             icon_expr="string:puzzle"
```

Comment:
Is there no trash icon?

Reply:
done in https://github.com/plone/Products.CMFPlone/pull/4168/commits/1ee283e2546e422b574fb8986eea92614fd3c808

Reply:
The one we should use is called `delete` (see https://pastanaga.io/icons/)

Reply:
My bad, I see we are using bootstrap icons now and `trash` should be okay.